server.js

```diff
@@ -2,36 +2,103 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import booksData from './data/books.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/books"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+const Book = mongoose.model('Book', {
+  bookID: {
+    type: Number,
+  },
+  title: {
+    type: String,
+  },
+  authors: {
+    type: String,
+  },
+  average_rating: {
+    type: Number,
+  },
+  isbn: {
+    type: String,
+  },
+  isbn13: {
+    type: String,
+  },
+  language_code: {
+    type: String,
+  },
+  num_pages: {
+    type: Number,
+  },
+  ratings_count: {
+    type: Number,
+  },
+  text_reviews_count: {
+    type: Number,
+  },
```

Comment:
It will be safer for you if you add required: true to some fields. this will prevent from creating empty entries in the database ;)

text_reviews_count is the last field of the model so you shouldn't add coma after it (line 41)